cmd/ceremony/crl_test.go

```diff
@@ -0,0 +1,53 @@
+package main
+
+import (
+	"crypto/ecdsa"
+	"crypto/elliptic"
+	"crypto/rand"
+	"crypto/x509"
+	"crypto/x509/pkix"
+	"math/big"
+	"testing"
+	"time"
+
+	"github.com/letsencrypt/boulder/test"
+)
+
+func TestGenerateCRL(t *testing.T) {
```

Comment:
I mainly agree we should be more granular here, but I think it makes sense to lump all of the time bounds checking into a single test, since it is all logically related and otherwise we end up with a bunch of boilerplate for no particular reason. I've split the time bounds checks and the general 'does it work' check into two tests.